src/Evolution/Initialization/Tags.hpp

```diff
@@ -3,22 +3,80 @@
 
 #pragma once
 
+#include <array>
 #include <cmath>
 #include <string>
+#include <vector>
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Options/Options.hpp"
 #include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
 namespace Initialization {
+namespace OptionTags {
+struct FuncOfTimeFromFile {
```

Comment:
`FuncOf...` -> `FunctionOf...` If the option parser gives you grief about name length you can specify a shorter name using the `name()` function